.github/workflows/warm-up-vercel.yml

```diff
@@ -29,4 +28,4 @@ jobs:
             curl --output /dev/null --silent --write-out "@curl-format.txt" "${BASE_URL}${PATHNAME}"
           done
         env:
-          BASE_URL: ${{ github.event.deployment_status && github.event.deployment_status.target_url || 'https://blockprotocol.org' }}
+          BASE_URL: ${{ github.event.deployment.environment != 'production' && github.event.deployment_status.target_url || 'https://blockprotocol.org' }}
```

Comment:
Calling `github.event.deployment.environment` seems to be working on GitHub even if `github.event.deployment` is `undefined` (I verified this by using `github.event.deployment-check.environment` temporarily).